### DISCUSSION TITLE
هل يصح كتابة المصطلح بالانجليزية بجانب الترجمة للمعاني الممكن ان تكون غير مفهومة ؟

### DISCUSSION THREAD

**Discussion by zeyadmoamen1210** [vuejs-translations/docs-ar#67] [سؤال] [answered]
مثال ان يتم اضافة الكلمات الانجليزية بجانب الترجمة بهذا الشكل 
"كما تري , defineAsyncComponent يقبل وظيفة أداة التحميل (Loader Function) والتي ترجع وعداً (Promise) "
 أم ان الترجمة يجب ان تكون عربية بشكل كامل ؟

**Comment by boussadjra** [comment, marked as answer]
أسماء المتغيرات و ما توفره الواجهة البرمجية من مصطلحات يجب أن يحتفظ به كما هو، و لكن يجب تعريفه مثلا `defineAsyncComponent `  هو اسم يشير إلى دالة فنقول "الدالة defineAsyncComponent  تقبل دالة تحميل  (Loader Function) كدالة رد نداء و التي تعيد لنا وعدا (Promise)  "
مثال آخر، نقول المتغير count تكون قيمته 1 بعد النقر على الزر 
بالنسبة للكلمات التقنية يمكن إضافة المصطلح الانجليزي بين قوسين في أول استعمالاتها، مثلا نقول "الخاصيات (props) تمرر إلى المكون الابن ..." ثم بمجرد أن تحس أن القارئ تعود على المصطلح العربي و ربطه بما يقابله من الانجليزي.
